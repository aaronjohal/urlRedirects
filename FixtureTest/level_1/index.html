<img src="images/foo.gif" />