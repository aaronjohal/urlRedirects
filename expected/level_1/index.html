<img src="http://news.imgbbc.co.uk/images/foo.gif" />